### ISSUE TITLE
escape colors on stdout

### ISSUE THREAD

**Issue by izelnakri** [doowb/ansi-colors#27]
when I test a cli app and use node.js child_process module to spawn a command and assert against the stdout out, chalk correctly escapes the colors, ansi-colors doesn't. 

In other words this library is not suitable for testing CLI packages, libraries. Please check how chalk handles it.

**Comment by doowb** [comment]
> Please check how chalk handles it.

Please read the documentation and update your CLI package: https://github.com/doowb/ansi-colors#toggle-color-support

`chalk` uses `supports-color` so you can do the following:

```js
const colors = require('ansi-colors');
colors.enabled = require('supports-color').stdout;
```

**Comment by izelnakri** [comment]
I was able to disable ansi colors for one of my libraries and it worked thanks. 

For another complex cli library I had, I couldn't use this library because there are tests per each build function and inside those functions I do ansi coloring. I had to still disable colors *per test case*, which isnt very ideal, thus had to use chalk again. 

Ava has a way to require modules before tests run however ansi-colors.enabled = false did not work for that case either. My suggestion is to disable coloring by default when NODE_ENV is test, if they provide ANSI_COLORS = true, then we can toggle off that default behavior.
